File: dbfit-java/core/src/main/java/dbfit/fixture/Execute.java
package dbfit.fixture;

import dbfit.api.DBEnvironment;
import dbfit.api.DbEnvironmentFacade;
import dbfit.api.DbEnvironmentFactory;
import dbfit.util.FitNesseTestHost;

import fit.Fixture;
import fit.Parse;

public class Execute extends Fixture {
    private String statementText;
    private DbEnvironmentFacade dbEnvironment;

    public Execute() {
        this(DbEnvironmentFactory.getDefaultEnvironment(), null);
    }

    public Execute(DBEnvironment env, String statement) {
        this.statementText = statement;
        this.dbEnvironment =
            new DbEnvironmentFacade(env, FitNesseTestHost.getInstance());
    }

    public void doRows(Parse rows) {
        try {
            dbEnvironment.runCommandWithBoundSymbols(getStatementText());
        } catch (Throwable e) {
            throw new Error(e);
        }
    }

    private String getStatementText() {
        if (statementText == null) {
            statementText = args[0];
        }
        return statementText;
    }
}

File: dbfit-dotnet/core/src/fixture/Execute.cs
using System;
using System.Collections.Generic;
using System.Text;
using fit;
using System.Data;
using System.Data.Common;
namespace dbfit.fixture
{
    public class Execute:fit.Fixture
    {
        private IDbEnvironment environment;
        private String statement;
        public Execute()
        {
            environment = DbEnvironmentFactory.DefaultEnvironment;
        }
        public Execute(IDbEnvironment environment, String statement)
        {
            this.environment = environment;
            this.statement = statement;
        }
        public override void DoRows(Parse rows)
        {
            if (String.IsNullOrEmpty(statement))
                statement = Args[0];
            using (DbCommand dc = environment.CreateCommand(statement, CommandType.Text))
            {
                if (dbfit.util.Options.ShouldBindSymbols()) 
                    environment.BindFixtureSymbols(dc);
                dc.ExecuteNonQuery();
            }
        }
    }
}
